### PR TITLE
Adds a twitter widget embed as a contentElement

### PR DESCRIPTION
**Pull request by UsmanMuhammad**
REDMINE-19882

This adds a Twitter widget embed as an addition to `contentElements`
A prescript is loaded to initiate Twitter widget, after the widget has
been loaded it uses the `twttr.widgets.createTweetEmbed` to create
a Twitter embed element.
Twitter embed currently takes a tweet-ID, and options to either hide
conversation or hide media like pictures, videos, etc.

### CODE REVIEW

**Review by tf — COMMENTED** [review]
This works well already! I mainly see room for improvement in the way tweets are displayed before consent has been given or while the script is still loading. Maybe we can display some sort of app shell - like a white card that already resembled the twitter card. This could also improve display in the editor before a tweet ID has been assigned.

Maybe we can discuss some options in a call once you are back later this week.

**Comment by tf** [comment]
It would also be nice to set up the storybook integration for the element so it can be included in the Percy CI step.

File: entry_types/scrolled/package/src/contentElements/twitterEmbed/editor.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import {editor} from 'pageflow-scrolled/editor';
+import {TextInputView, CheckBoxInputView} from 'pageflow/ui';
+import pictogram from './pictogram.svg'
+
+editor.contentElementTypes.register('twitterEmbed', {
+  pictogram,
+  category: 'media',
+  supportedPositions: ['inline', 'sticky', 'left', 'right', 'wide', 'full'],
```
[the remark this answers is not in the document]
Comment (UsmanMuhammad) [reply entry_types/scrolled/package/src/contentElements/twitterEmbed/editor.js:8]: Removed the 'full' position.

**Comment by UsmanMuhammad** [comment]
Closing this PR due to rebasing issues, the new PR can be found #1852

**Pull request closed without merging**